tests/next_tests/otf_tests/test_cpp_gen.py

```diff
@@ -107,3 +107,54 @@ def test_render_function_call_buffer(function_buffer_example):
     )
     expected = format_source("cpp", """example(get_arg_1(), get_arg_2())""", style="LLVM")
     assert rendered == expected
+
+
+@pytest.fixture
```

Comment:
I just used the above tests and extended them with my feature, but it doesn't seem super useful...

Reply:
These tests look like the suffer from similar problems as the ffront lowering tests. Let's keep them for now and revisit when they start to bug us.